searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizer/RCNN.java

```diff
@@ -0,0 +1,33 @@
+package org.jalau.at18.searchobject.model.objectrecognizer.recognizer;
+
+import org.jalau.at18.searchobject.model.objectrecognizer.recognizertypes.MatchInfo;
+import org.jalau.at18.searchobject.model.objectrecognizer.recognizertypes.rcnn.RcnnDetection;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
```

Comment:
add documentation to the model

Reply:
@ferAguilar02 erase the upload images

Reply:
I have added documentation and removed all images